submissions/DaniaB24/js/app.js

```diff
@@ -0,0 +1,120 @@
+const cardsOfFriends = document.getElementById("allHumans");
+const searchBar = document.getElementById("search");
+
+let filteredFriends;
+let allFriends;
+let sortedFriends;
+let friends;
+fetch("https://randomuser.me/api/?results=10")
+  .then((response) => response.json())
+  .then((users) => {
+    friends = users.results;
+    renderFriends(friends);
+    displayUsers(friends);
+    document.addEventListener("click", (event) =>
+      filterSetting(event, friends)
+    );
+  });
+
+function filterSetting(event, friends) {
+  renderFriends(friends);
+  filterByGender(friends);
+
+  if (event.target.closest(".filter_menu"))
+    sortFriends(event.target, filteredFriends);
+}
+function renderFriends(friends) {
+  filterByGender(friends);
```

Comment:
Why do you execute the filter on the initial render? 🤔

Reply:
it's like a crutch )) I will fix it